extensions/mysql/index.js

```diff
@@ -77,7 +76,74 @@ class MySQLExtension extends cli.Extension {
     }
 
     createUser(ctx, dbconfig) {
-        const randomPassword = crypto.randomBytes(10).toString('hex');
+        // Generate random MySQL password
+        const generateRandomPassword = () => {
+            /**
+             * LCG Random
+             * @example cosnt random = (lcgRandom())
```

Comment:
Was this supposed to say `const` instead of `cosnt`?

Reply:
Oh! Sorry! Just a spelling error.